code/modules/projectiles/guns/ego_gun/waw.dm

```diff
@@ -218,10 +221,10 @@
 	special = "This weapon fires extremely slowly. \
 		This weapon pierces all targets."
 	ammo_type = /obj/item/ammo_casing/caseless/ego_magicpistol
-	fire_delay = 24
+	fire_delay = 12
```

Comment:
No way, undocumented buffs.

Reply:
Unintentionally pushed due to leftovers from another branch....